libsolidity/interface/CompilerStack.h

```diff
@@ -116,6 +116,12 @@ class CompilerStack: boost::noncopyable
 		m_optimizeRuns = _runs;
 	}
 
+	/// Sets the list of requested contract names. Names are cleared iff @a _contractNames is missing.
```

Comment:
What is the effect of the names being cleared?
Perhaps rather: `Defaults to "all contracts" (empty set)`